src/ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.4.19f1
+m_EditorVersion: 2017.4.17f1
```

Comment:
Can you take this change back please? It seems odd to downgrade the unity version.

Reply:
done